docs/paper/admin/misc/paper-bug-fixes.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: /misc/paper-bug-fixes
+---
+
+# Paper Bug Fixes
+
+Paper fixes gameplay and mechanics inconsistencies. The most prevalent fixes being TNT Duplication and Bedrock Breaking.
+
+## Vanilla Bug Fixes
+
+Paper fixes many vanilla bugs that were not intended by Mojang. These bugs are patched to fix behavior or prevent abuse and
+instability on the server. Many of our fixes are configurable as we understand that some servers may want to keep the
+vanilla behavior. You will find these configuration options in the [global configuration](/docs/paper/admin/reference/configuration/global-configuration.mdx)
+and the [world configuration](/docs/paper/admin/reference/configuration/world-configuration.mdx).
+
+### What is intended behavior vs a bug?
+
+When an issue is reported to us, we check Mojang's issue tracker. If the problem has been reported there, then we
+check to see if it:
+
+1) Has been confirmed as a bug
+2) Has an assigned priority to it
+
+If it meets these two criteria then we will accept changes to fix the bug as it can take a long time for Mojang to fix 
+them (Sometimes years). If an issue gets declined by Mojang, we normally do not "fix" it as it is intended behaviour.
+
+## Duplication Bugs
+
+Paper Patches TNT, Carpet, Rail and Gravity Block (Sand, Gravel, etc.) Duplication bugs. These bugs are not intended and 
+are patched to prevent abuse.
+
+Because TNT duping is considered a form of automated mining and not a resource dupe, we have provided an option to 
+restore it. This, undesirably, also re-enables carpet and rail duping, which normally we would not provide a config for, 
+but it's the same bug for those, so we have no choice. However, the config option is as follows:
+
+```yaml
+unsupported-settings:
+  allow-piston-duplication: true
+```
+
+:::danger[Sand and Gravity Blocks]
+
+**We will not and will never support sand and gravity block duping**. This is a form of resource duping and is **not** supported by Paper.
+
+:::
+
+## Block Breaking
+
+We also fix the ability to break Bedrock and End Portal frames. We do also provide a config option to restore this
```

Comment:
```suggestion
We also fix the ability to break bedrock and end portal frames. We do also provide a config option to restore this
```

Reply:
It's possible these are proper nouns as the name of blocks in game so capitalization would be appropriate.